### PR TITLE
DOC: add #212 to the Changelog

### PR DESCRIPTION
**Pull request by adswa**
This is a leftover from #212.

### CODE REVIEW

**Comment by sappelhoff** [comment]
@adswa I took the liberty to make the docstr one line (see acda172), which is what Mainak wanted, and which I oversaw before merging your PR. (committed directly to your branch)

**Review by hoechenberger — APPROVED** [review]
LGTM, feel free to merge once Circle comes back green

**Comment by adswa** [comment]
> @adswa I took the liberty to make the docstr one line (see [acda172](https://github.com/autoreject/autoreject/commit/acda17207dca62d90bc72ec1b4cecbd1fc21e1c8)), which is what Mainak wanted, and which I oversaw before merging your PR. (committed directly to your branch)

Thanks!

File: doc/whats_new.rst [hunk 1]
```diff
@@ -17,6 +17,8 @@ Changelog
 
 - Make plotting of reject log more flexible by allowing to change the orientation of plot, option to supress the immediate display of the plot and returning the plot figure for further modification or saving in :meth:`autoreject.RejectLog.plot`, by `Marcin Koculak`_ in `#152 <https://github.com/autoreject/autoreject/pull/152>`_
 
+- Adjust usage of an internal method to a parametrization change in mne, by `Adina Wagner`_ in `#212 <https://github.com/autoreject/autoreject/pull/212>`_.
```
Comment (jasmainak) [inline doc/whats_new.rst:20]: I think this should go in the Bug section.

Sorry for so many requests !

**Comment by jasmainak** [comment]
I fixed `whats_new`. Thanks a ton @adswa for your first contribution to `autoreject`!!

**Comment by hoechenberger** [comment]
Welcome to `autoreject` – where your PRs get automatically rejected until everything's green! 😅👌😁☠️

**Comment by jasmainak** [comment]
that joke was waiting to be made :) :)

**Pull request merged**